openmetadata-ui/src/main/resources/ui/src/pages/AddServicePage/AddServicePage.component.tsx

```diff
@@ -43,7 +43,7 @@ import { showErrorToast } from '../../utils/ToastUtils';
 
 const AddServicePage = () => {
   const { serviceCategory } = useParams<{ [key: string]: string }>();
-  const [newServiceData, setNewServiceData] = useState<DataObj>();
+  const [newServiceData, setNewServiceData] = useState<ServicesUpdateRequest>();
```

Comment:
Not sure if `ServicesUpdateRequest` would be the correct type here, can you please confirm?

Reply:
Yes it's correct for now, We need to update all this with proper types as we discussed